checker/bin/wpi-many.sh

```diff
@@ -85,8 +85,8 @@ if [ "${has_java_home}" = "yes" ] && [ ! -d "${JAVA_HOME}" ]; then
     exit 1
 fi
 
+java_version=$("${JAVA_HOME}"/bin/java -version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1)
 if [ "${has_java_home}" = "yes" ]; then
-    java_version=$("${JAVA_HOME}"/bin/java -version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1)
```

Comment:
If this is moved outside the test of `${has_java_home}`, then won't this command always fail when the `$JAVA_HOME` environment variable isn't specified? That seems undesirable, but maybe I just don't understand the motivation for this change.

Reply:
Good point.  I reverted this change.